### DISCUSSION TITLE
Open data for GWAS

### DISCUSSION THREAD

**Discussion by hammer** [sgkit-dev/sgkit#1144] [General]
I will organize by species. This box can be for cross-species resources or synthetic data.

- [GenomeArk](https://vgp.github.io/) looks to be a great source of open data genome reference data.
- [GWAS Atlas: a curated resource of genome-wide variant-trait associations in plants and animals](https://pubmed.ncbi.nlm.nih.gov/31566222/) (2020)
- [Naturalgwas: An R package for evaluating genomewide association methods with empirical data](https://www.ncbi.nlm.nih.gov/pubmed/29673087) (2018): code at [bcm-uga/NaturalGWAS](https://github.com/bcm-uga/NaturalGWAS). Could allow us to simulate phenotypes with desired properties useful for benchmarking.
- On the UKB mailing list [Ben Cairns](http://users.ox.ac.uk/~clme1073/) says he is working with the UKB team to make a synthetic data set available. For now we could use https://github.com/spiros/tofu.
- [Harnessing genomics to fast-track genetic improvement in aquaculture](https://www.nature.com/articles/s41576-020-0227-y) (2020): awesome review paper; table 1 has a list of genome resources available for acquatic animals important to the food supply. Looks like oysters are the way to go and [Ross Houston](https://www.ed.ac.uk/profile/ross-houston) at Edinburgh is publishing a lot of the papers.

**Comment by hammer** [comment]
### Chicken
- [Disrupting The Pecking Order: Blue Apron Cofounder Raises $10 Million In Funding To Breed A Better Chicken](https://www.forbes.com/sites/chloesorvino/2020/07/22/blue-apron-cofounder-raises-10-million-in-funding-to-breed-a-better-chicken/#1efa46181f0d) indicates that Cobb and Aviagen (Ross) are the dominant genetics companies in the US chicken market, but Cooks Venture wants to join them.

Some important papers in the history of chicken genomics
- [A consensus linkage map of the chicken genome](https://pubmed.ncbi.nlm.nih.gov/10645958/) (2000)
- [Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution](https://pubmed.ncbi.nlm.nih.gov/15592404/) (2004)
- [A genetic variation map for chicken with 2.8 million single-nucleotide polymorphisms](https://pubmed.ncbi.nlm.nih.gov/15592405/) (2004)
- [A New Chicken Genome Assembly Provides Insight into Avian Genome Structure](https://pubmed.ncbi.nlm.nih.gov/27852011/) (2017)
- [Genome-Wide Population Genetic Analysis of Commercial, Indigenous, Game, and Wild Chickens Using 600K SNP Microarray Data](https://www.frontiersin.org/articles/10.3389/fgene.2020.543294/full) (2020): looking for selection signals in 1,254 samples from 15 chicken populations genotyped with the 600K Affymetrix Axiom Chicken Genotyping Array. Data is [available on figshare](https://figshare.com/s/efe1bbbf60078f2f08aa?file=23960792) in PLINK's ped/map 2-file format.

**Comment by hammer** [comment]
### Maize
- [Genome-wide Association Study Dissects the Genetic Architecture of Oil Biosynthesis in Maize Kernels](https://pubmed.ncbi.nlm.nih.gov/23242369/) (2013): big GWAS from the National Maize Improvement Center of China
- [MaizeGDB](https://www.maizegdb.org/) seems to organize reference genomes
- [Panzea](https://www.panzea.org) seems to organize variation data
- [Identification of key tissue-specific, biological processes by integrating enhancer information in maize gene regulatory networks](https://www.biorxiv.org/content/10.1101/2020.06.16.155481v1) (2020) generates some data similar to ENCODE and GTEx.
- [Natural variations and genome-wide association studies in crop plants](https://pubmed.ncbi.nlm.nih.gov/24274033/) (2014): nice review paper that highlights rice and maize as the two crops with the most robust GWAS data.

**Comment by hammer** [comment]
### Potato
- [NSF Potato Genome Project](http://potatogenome.berkeley.edu/nsf5/)
- [Genome sequence and analysis of the tuber crop potato](https://pubmed.ncbi.nlm.nih.gov/21743474/) (2011)
- Potato genomes are polyploid!

**Comment by hammer** [comment]
### Wheat
- [Limited haplotype diversity underlies polygenic trait architecture across 70 years of wheat breeding](https://www.biorxiv.org/content/10.1101/2020.09.15.296533v1) (2020): array and sequencing data on over 500 individuals derived from 16 founders and 47 phenotypes. The individuals are from the [NIAB diverse MAGIC project](http://mtweb.cs.ucl.ac.uk/mus/www/MAGICdiverse/index.html), and there's some code at https://github.com/michaelfscott/DIVERSE_MAGIC_WHEAT.

**Comment by hammer** [comment]
### *Arabidopsis thaliana*
- [1001 Genomes](http://1001genomes.org/) has VCF data for 1,135 strains available for download with no login necessary. [1,135 Genomes Reveal the Global Pattern of Polymorphism in Arabidopsis thaliana](https://pubmed.ncbi.nlm.nih.gov/27293186/) (2016) appears to be the associated publication.
- There's also a [AraPheno](https://arapheno.1001genomes.org/) project that collects phenotypes, and an [AraGWAS Catalog](https://aragwas.1001genomes.org/#/). [The AraGWAS Catalog: a curated and standardized Arabidopsis thaliana GWAS catalog](https://pubmed.ncbi.nlm.nih.gov/29059333/) (2018) has more details.

**Comment by hammer** [comment]
### Hummingbird
- [De novo PacBio long-read and phased avian genome assemblies correct and add to reference genes generated with intermediate and short reads](https://pubmed.ncbi.nlm.nih.gov/29020750/) (2017)
- [Single-molecule, full-length transcript sequencing provides insight into the extreme metabolism of the ruby-throated hummingbird Archilochus colubris](https://pubmed.ncbi.nlm.nih.gov/29618047/) (2018)"

**Comment by hammer** [comment]
### Dog
- [Complex disease and phenotype mapping in the domestic dog](https://www.ncbi.nlm.nih.gov/pubmed/26795439) (2016) has 181 MB of data available from https://datadryad.org/stash/dataset/doi:10.5061/dryad.266k4. The bigsnpr paper uses this data.
- [Genome-wide association analysis in dogs implicates 99 loci as risk variants for anterior cruciate ligament rupture](https://www.ncbi.nlm.nih.gov/pubmed/28379989/) (2017) has 16 MB of data available from https://datadryad.org/stash/dataset/doi:10.5061/dryad.8kk06.
- [Imputation of canine genotype array data using 365 whole-genome sequences improves power of genome-wide association studies](https://www.ncbi.nlm.nih.gov/pubmed/31525180) (2019) has 11 GB (!!!) of data available from https://datadryad.org/stash/dataset/doi:10.5061/dryad.jk9504s.

**Comment by hammer** [comment]
### Rice
- [The impact of transposable elements on the structure, evolution and function of the rice genome](https://www.ncbi.nlm.nih.gov/pubmed/31797393) (2020)
- [BrAPI-an application programming interface for plant breeding applications](https://www.ncbi.nlm.nih.gov/pubmed/30903186) (2019)
- [Analyzing 3024 rice genomes characterized by DeepVariant](https://cloud.google.com/blog/products/data-analytics/analyzing-3024-rice-genomes-characterized-by-deepvariant) (2019): Google blog post on re-calling variants for the Rice 3K data with DeepVariant (vs. GATK)
- [Genomic variation in 3,010 diverse accessions of Asian cultivated rice](https://www.ncbi.nlm.nih.gov/pubmed/29695866) (2018)
- [Open access resources for genome-wide association mapping in rice](https://www.ncbi.nlm.nih.gov/pubmed/26842267) (2016): http://ricediversity.org
- [RiceVarMap: a comprehensive database of rice genomic variations](https://www.ncbi.nlm.nih.gov/pubmed/25274737) (2015): http://ricevarmap2.ncpgr.cn
- [SNP-Seek database of SNPs derived from 3000 rice genomes](https://www.ncbi.nlm.nih.gov/pubmed/25429973) (2015): https://snp-seek.irri.org
- [The 3,000 rice genomes project](https://www.ncbi.nlm.nih.gov/pubmed/24872877) (2014)

**Comment by hammer** [comment]
### Mosquito
- [Genome variation and population structure among 1,142 mosquitoes of the African malaria vector species Anopheles gambiae and Anopheles coluzzii](https://www.biorxiv.org/content/10.1101/864314v2) (2020): the flagship paper from the Anopheles gambiae 1000 Genomes Consortium (Ag1000G). Alistair Miles wrote [scikit-allel](https://github.com/cggh/scikit-allel) to analyze this data. The data is [available for download](https://www.malariagen.net/resource/27), and they've set up a GitHub repo for issues related to the data: [malariagen/vector-public-data](https://github.com/malariagen/vector-public-data). Note there's no phenotype data in this project.
- [Genetic diversity of the African malaria vector Anopheles gambiae](https://www.ncbi.nlm.nih.gov/pubmed/29186111) (2017): precursor to the above paper.

**Comment by hammer** [comment]
### Cattle
- [Analysis of a large dataset reveals haplotypes carrying putatively recessive lethal and semi-lethal alleles with pleiotropic effects on economically important traits in beef cattle](https://www.ncbi.nlm.nih.gov/pubmed/30836944) (2019): 132,725 cattle genotyped at around 40k SNPs. Data comes from the Irish Cattle Breeding Federation (ICBF) as part of their [Beef Data and Genomics Programme](https://www.agriculture.gov.ie/beefschemes/) that is going to genotype 1.3M cattle!
- [Council on Dairy Cattle Breeding](https://www.uscdcb.com/what-we-do/genomics/) (CDCB) has [apparently](https://twitter.com/juanonani/status/1237401795779014659) genotyped 4M cows!
- [Powerful detection of polygenic selection and environmental adaptation in US beef cattle populations](https://www.biorxiv.org/content/10.1101/2020.03.11.988121v3) (2020): data from Red Angus Association of America, American Simmental Association, and American Gelbvieh Association.
- [Building a livestock genetic and genomic information knowledgebase through integrative developments of Animal QTLdb and CorrDB](https://pubmed.ncbi.nlm.nih.gov/30407520/) (2019)

**Comment by hammer** [comment]
### Bean
- [Genetic Associations in Four Decades of Multienvironment Trials Reveal Agronomic Trait Evolution in Common Bean](https://www.genetics.org/content/215/1/267) (2020): Cooperative Dry Bean Nursery (CDBN) (*Phaseolus vulgaris*) has 21 CDBN phenotypes and genotypic data (1.2 million SNPs) for 327 CDBN genotypes

**Comment by gregorgorjanc** [comment]
Here is one public dataset for pigs [A Common Dataset for Genomic Analysis of Livestock Populations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3337471/) - not huge, but better than nothing. The challenge with agricultural datasets is that public funding doesn't provide sufficient scope for large scale genotyping and phenotyping so most data is actually owned by breeding companies.

**Reply by jeromekelleher** [reply]
Recent synthetic data generated by [HAPNEST](https://academic.oup.com/bioinformatics/article/39/9/btad535/7255913) with 1 million samples available for download in plink format here: https://www.ebi.ac.uk/biostudies/studies/S-BSST936

**Comment by timothymillar** [comment]
### Sheep

- [Estimating quantitative genetic parameters in wild populations: a comparison of pedigree and genomic approaches](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4149785) (2014). (https://datadryad.org/stash/dataset/doi:10.5061/dryad.367s2). Pedigree of 6740 individuals with genotypes for 3973 individuals (37,037 SNPs) in PLINK format. There are a range of phenotypes. Data is on [Dryad].

**Comment by gregorgorjanc** [comment]
A collection of various databases with AG data (of various kinds!) https://www.agbiodata.org/
